src/openrct2/scenario/ScenarioRepository.cpp

```diff
@@ -48,7 +47,7 @@ static int32_t ScenarioCategoryCompare(int32_t categoryA, int32_t categoryB)
         return -1;
     if (categoryB == SCENARIO_CATEGORY_BUILD_YOUR_OWN)
         return 1;
-    return Math::Sign(categoryA - categoryB);
+    return (categoryA < categoryB) ? -1 : 1;
```

Comment:
Math::Sign would return 0 when the input was 0.

Reply:
OP mentioned that in the comment, it's being checked at the top of this function. I'm not really in favour of replacing the function with a ternary operator though. I'd rather see an easier-to-read branch instead, like the other statements above.

Reply:
That is handled at the first line of this function